src/lang/cpp/mod.rs

```diff
@@ -464,8 +460,8 @@ fn transform_into_class(
         }
     }
 
-    let line_begin = field_list.children.iter().next()?.span?.0;
-    let line_end = field_list.children.iter().last()?.span?.0;
+    let line_begin = field_list.children.first()?.span?.0;
+    let line_end = field_list.children.first()?.span?.0;
```

Comment:
The original had `iter().last()` here, while the new has `first()`; was this intentional? Is the collection known to only have one element in it?

Reply:
You're right. This should be getting the last, not first. Will fix